### ISSUE TITLE
Cumberland Athletics Website

### ISSUE THREAD

**Issue by Alexd1204** [CHS-WBL-2022/WBL22-Community#5]
A website that holds information on Cumberland athletics, including a detailed webpage that has dropdown menus for sports that will include links to all the sports the school has to offer for boys and girls. The links for each team will include schedules, rosters, news, and social media pages. The site will also include an events page(schedule/calendar) with every sports upcoming events. There will be more dropdowns that will feature the necessary paperwork that need to be completed by athletes so it can be accessed by students and parents. The page could also include a shop page for Cumberland Clippers merchandise. The website will include any other necessary links/information that is needed and wanted by our community program/member. Also the website will be loaded with photos from our schools sports teams. The website can also include the schools sponsors and sports team individual sponsors. Another feature that is seen on college athletic sites similar to the one I have in mind include athlete of the week for men's and women's sports that will highlight their outstanding performances.

**Comment by KennyRoch** [comment]
ye

**Comment by nlefebvre24** [comment]
tl dr but good idea
